src/contexts/authContext.js

```diff
@@ -73,19 +92,45 @@ const AuthProvider = ({ children }) => {
           });
           console.log(createdUser);
           navigate("/");
-          localStorage.setItem("JWT_TOKEN", encodedToken);
+          toast.success("Succesfully Signed Up & In!", {
+            position: "bottom-center",
+            autoClose: 3000,
+            hideProgressBar: true,
+            closeOnClick: true,
+            pauseOnHover: true,
+            draggable: true,
+            progress: undefined,
+          });
         }
       } catch (error) {
         console.log(error);
         if (error.response.status === 422) {
           userDispatch({ type: "USER_ALREADY_EXISTS" });
+          toast.warning("User Already Exists", {
+            position: "bottom-center",
+            autoClose: 3000,
+            hideProgressBar: true,
+            closeOnClick: true,
+            pauseOnHover: true,
+            draggable: true,
+            progress: undefined,
+          });
         }
       }
     }
   };
   const logoutUser = () => {
     navigate("/");
     userDispatch({ type: "USER_LOGOUT" });
```

Comment:
suggestion: store the dispatch type in constants or variables, do not hardcode it.

Reply:
yes, hardcoding dispatch type is not a good practise